R/source.r

```diff
@@ -18,7 +18,8 @@ source_one <- function(file, envir = parent.frame()) {
   stopifnot(file.exists(file))
   stopifnot(is.environment(envir))
 
-  lines <- readLines(file, warn = FALSE)
+  lines <- readLines(file, warn = FALSE, encoding = "UTF-8")
+  lines <- enc2native(lines)
```

Comment:
I'm pretty sure this is necessary, because I just reproduced this to confirm.  

1. Let's say we are working on a windows platform, whose native encoding is not `utf-8`,
1. If we have a function (e.g., `futf8()`) that contains nonASCII chars in some package ,
1. Without `lines <- enc2native(lines)`, `devtools::load_all()` performs good and the function `futf8()` works well,
1. But, if you `print(futf8)`, you will find mess in the console,
1. It will be solved by adding this line.

### without the line

![image](https://cloud.githubusercontent.com/assets/8368933/19997461/abc1de0a-a2a1-11e6-8e05-534803eef315.png)

### with this line

![image](https://cloud.githubusercontent.com/assets/8368933/19997471/c3abc828-a2a1-11e6-95e2-4be053579f15.png)